disnake/threads.py

```diff
@@ -332,6 +332,14 @@ def created_at(self) -> datetime.datetime:
         """
         return self.create_timestamp or snowflake_time(self.id)
 
+    @property
+    def flags(self) -> ChannelFlags:
+        """:class:`ChannelFlags`: The channel flags for this channel.
+
+        .. versionadded:: 2.6
```

Comment:
```suggestion
        .. versionadded:: 2.5
```
also duplicated documentation in class docstring, that one should be removed

Reply:
https://github.com/DisnakeDev/disnake/pull/547/commits/73adfc9a7d5d67329151450f21b3288079efad4b